resources/message/en/gridview.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'gridview.summary' => 'Showing <b>{pageCount}</b> of <b>{totalCount}</b>',
```

Comment:
How about the following naming?

```suggestion
    'gridview.summary' => 'Page <b>{currentPage}</b> of <b>{totalPages}</b>',
```

Reply:
done.